src/main/java/org/myrobotlab/framework/Inbox.java

```diff
@@ -166,11 +166,16 @@ public Message getMsg() throws InterruptedException {
             // correct but this works for
             // null data now
             {
-              returnContainer = null;
+              returnContainer[0] = null;
             } else {
               returnContainer[0] = msg.data[0]; // transferring
               // return data !
             }
+            /*

```

Comment:
please delete this commented out code if it's not used.

Reply:
fixed